.changelog/1382.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+config: Add ability to define internal variables and compose variables together via templating
```

Comment:
Nitpick, we may want to turn this one into a bold heading style (see 0.3 or 0.2 or w/e changelog) but we can do that human-powered when we do a major release?

Reply:
You should be able to use markdown here and go-changelog will respect it, so adding the `*` _should_ be fine to do here if we want to bold it

Reply:
Cool, yeah worth considering, I think this is a big feature.